server/db/db.js

```diff
@@ -145,6 +145,19 @@ columns.addresses = {
   country: Sequelize.STRING
 }
 
+columns.athleteProfiles = {
```

Comment:
Might need to add more columns to this in the future, but can manually add columns to the prod DB if necessary